### DISCUSSION TITLE
Wiki and Blogpost "Setting up our wiki"

### DISCUSSION THREAD

**Discussion by qvalentin** [FileFighter/filefighter.github.io#11] [General]
Hey,
comment your feedback about our "Setting up our wiki" page and our wiki page from your peer-review here:

**Comment by FranzJosef2000** [comment]
Hi FileFighter Team,
Your SRS is very advanced and described in detail, which leads to the conclusion that many things have already been thought through and clarified.

Your UCD is clear and well structured, especially that you do not display the functions that are not implemented outside the box, but in a different color, which is not in accordance with the convention, but shows which function is grouped where.

We are very excited about the further progress,
Out-Of-Memory team

**Comment by qvalentin** [comment]
Dear Out-Of-Memory team,

thank you for your positive feedback.

FileFighter Team

**Comment by DSinMeliodas** [comment]
Dear FileFighter Team,
I have not much to say eighter everything has allready been said. Your SRS and UCD are both detailed, clear and well structured just like all of your current blog posts.

I'm very excited and looking forward towards your next blog,
with best regards the Futurum team.
